test/utils/http.js

```diff
@@ -132,8 +99,17 @@ class HttpTestUtils {
       throw Error('invalid type supplied');
     }
 
-    r = (code === STATUS_CODES.OK || code === STATUS_CODES.BAD_REQUEST) ? r.expect('Content-Type', 'application/json; charset=utf-8') : r;
-    r = (code === STATUS_CODES.METHOD_NOT_ALLOWED) ? r.expect('Allow', allowedType) : r;
+    for (const h in headers) {
+      r = r.set(h, headers[h]);
+    }
+
+    if (code === STATUS_CODES.METHOD_NOT_ALLOWED) {
+      r = r.expect('Allow', allowedType);
+    }
+
+    if (code === STATUS_CODES.OK || code === STATUS_CODES.BAD_REQUEST) {
+      r = r.expect('Content-Type', /json/);
```

Comment:
Why the switch to the regex `/json/` instead of the string `'application/json; charset=utf-8'`?